docs/components/input-field/demo/base-demo.md

```diff
@@ -1,12 +1,17 @@
 ```hbs template
 <Form::Fields::Input
   @label='Label'
-  @hint='Type "input" into the field'
-  @error={{this.errorMessage}}
+  @hint='Hint text'
```

Comment:
Any reason to adjust the `hint` and `error` so that we can test the error state + hint state properly as part of this demo?

Reply:
Should be reverted now!